src/features/projectsV2/ProjectsMap/index.tsx

```diff
@@ -48,11 +49,17 @@ function ProjectsMap(props: ProjectsMapProps) {
         mapRef,
         e.point
       );
-      if (hoveredPlantLocation) setHoveredPl(hoveredPlantLocation);
+      if (
+        hoveredPlantLocation &&
+        hoveredPlantLocation?.hid !== selectedPl?.hid
+      ) {
+        setHoveredPl(hoveredPlantLocation);
+      } else {
+        setHoveredPl(null);
+      }
     },
-    [plantLocations]
+    [plantLocations, selectedPl]
   );
-
   const onClick = useCallback(
     (e) => {
       const selectedPlantLocation = getPlantLocationInfo(
```

Comment:
Currently, I cannot unset a sample plant location once selected, if I click on the containing plant location. The only way to do so is by clicking on a different plant location and then coming back to the first one.